packages/formation-react/src/components/SystemDownView/SystemDownView.jsx

```diff
@@ -20,10 +20,10 @@ class SystemDownView extends React.Component {
 }
 
 SystemDownView.propTypes = {
-  // first line of the system down message, required
+  // First line of the system down message
```

Comment:
So storybook will properly show the description in the `<ArgBlock>` or whatever it is called on the docs page

```suggestion
  /**
   * First line of the system down message 
   */
```